packages/dual-channel/docs/zh-tw/mannual.md

```diff
@@ -62,6 +62,17 @@ _圖 1-1：深咖啡色的方框是章節，淺咖啡色的方框則是區塊。
 
 大家可以參考[「報導者／左右互搏版型」測試資料 - Google 試算表](https://docs.google.com/spreadsheets/d/1Ppisv4HTZHYMp95umgCoADNuP1PSkL-t9na-5lRIqSY/edit#gid=2143095237)。
 
+#### 設定共用
+
+- 注意在修改資料時，不要改到有灰底色的儲存格。
```

Comment:
I just noticed that this line should be with data editing section. I'll fix it in `timeline` also.